src/elements/button/mini-button-group/mini-button-group.scss

```diff
@@ -0,0 +1,61 @@
+@use '../../core/styles/index' as sbb;
+
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
+@include sbb.box-sizing;
+
+:host {
+  display: block;
+
+  --sbb-mini-button-group-background: var(--sbb-color-white);
+  --sbb-mini-button-group-border-color: var(--sbb-color-cloud);
+  --sbb-mini-button-group-border-width: var(--sbb-border-width-2x);
+  --sbb-mini-button-group-border: var(--sbb-mini-button-group-border-width) solid
+    var(--sbb-mini-button-group-border-color);
+  --sbb-mini-button-group-gap: var(--sbb-spacing-fixed-2x);
+  --sbb-mini-button-group-min-height: var(--sbb-size-element-s);
+  --sbb-mini-button-group-padding-inline: var(--sbb-spacing-fixed-3x);
+}
+
+:host([negative]) {
+  --sbb-mini-button-group-background: var(--sbb-color-midnight);
+  --sbb-mini-button-group-border-color: var(--sbb-color-anthracite);
+}
+
+:host([size='s']) {
+  --sbb-mini-button-group-min-height: var(--sbb-size-element-xs);
+  --sbb-mini-button-group-padding-inline: var(--sbb-spacing-fixed-2x);
+}
+
+:host([size='l']) {
+  --sbb-mini-button-group-min-height: var(--sbb-size-element-m);
+}
+
+:host([size='xl']) {
+  --sbb-mini-button-group-min-height: var(--sbb-size-element-l);
+  --sbb-mini-button-group-padding-inline: var(--sbb-spacing-fixed-4x);
+}
+
+::slotted(sbb-divider) {
+  --sbb-divider-border-width: var(--sbb-border-width-1x);
+
+  padding-block: var(--sbb-spacing-fixed-1x);
+  padding-inline: var(--sbb-spacing-fixed-1x);
+  height: var(--sbb-size-icon-ui-small);
+}
+
+.sbb-mini-button-group {
```

Comment:
--sbb-mini-button-group-background is not applied (leads to wrong visual output in negative mode)